modules/gitops-using-argo-cd-instance-to-manage-cluster-scoped-resources.adoc

```diff
@@ -6,7 +6,7 @@
 [id="using-argo-cd-instance-to-manage-cluster-scoped-resources{context}"]
 = Using an Argo CD instance to manage cluster-scoped resources
 
-To manage cluster-scoped resources, update the existing `Subscription` object for the `gitops-title` Operator and add the namespace of the Argo CD instance to the `ARGOCD_CLUSTER_CONFIG_NAMESPACES` environment variable in the `spec` section.
+To manage cluster-scoped resources, update the existing `Subscription` object for the `{gitops-title}` Operator and add the namespace of the Argo CD instance to the `ARGOCD_CLUSTER_CONFIG_NAMESPACES` environment variable in the `spec` section.
```

Comment:
This is now the only instance in the assembly where it's marked up in the style of "`Red Hat OpenShift GitOps` Operator". All other instances are just "Red Hat OpenShift GitOps Operator" (no backtick markup).

Was the original typo maybe that the backticks should have instead been curly brackets, not that it should have both? I.e.:
```suggestion
To manage cluster-scoped resources, update the existing `Subscription` object for the {gitops-title} Operator and add the namespace of the Argo CD instance to the `ARGOCD_CLUSTER_CONFIG_NAMESPACES` environment variable in the `spec` section.
```
FWIW, when I check the rest of the repo, I find lots of:
```
{gitops-title} Operator
```
but no:
```
`{gitops-title}` Operator
```

Reply:
Yes, @adellape. I have removed the backticks from both the instances to ensure that we maintain consistency throughout the document. Kindly have a look and confirm the changes.

Reply:
Sending a gentle reminder to you @adellape so that this PR can be merged.